### PR TITLE
govuk style changes and copy tweaks

### PR DESCRIPTION
**Pull request by TheDancingClown**
https://app.asana.com/0/1200175839265057/1200311085430209
https://drive.google.com/file/d/1HRtgfNZXP4pbUqeLMtsBqRTVv9E6xTPf/view

Changes deadline panel to notification banner
Screenshot 2021-08-04 at 09 32 47

Change to inline radio buttons
Screenshot 2021-08-04 at 09 32 58

Removes header from question 8 and adds notification border to question 9
Screenshot 2021-08-04 at 09 33 12

### CODE REVIEW

File: app/views/qae_form/_options_question.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.govuk-radios
+.govuk-radios--inline
```
Comment (mauricioac) [inline app/views/qae_form/_options_question.html.slim:1]: @TheDancingClown I think it needs both `govuk-radios` and `govuk-radios--inline`

https://design-system.service.gov.uk/components/radios/

File: app/forms/award_years/v2022/qavs/qavs_step5.rb [hunk 2]
```diff
@@ -343,7 +338,7 @@ def qavs_step5
 
         options :local_assessment_verdict, "Local assessment outcome" do
           sub_ref "E 9"
-          classes "sub-question"
+          classes "govuk-notification-banner govuk-notification-banner__content"
```
Comment (mauricioac) [inline app/forms/award_years/v2022/qavs/qavs_step5.rb:341]: I dont think you can put the parent class and the child class together.
If this is the case, just leave the parent, cause the two together are not doing what they originally should be doing.

Or does it make a difference here?

Comment (TheDancingClown) [reply app/forms/award_years/v2022/qavs/qavs_step5.rb:341]: It did make a difference. Just using the parent has a blue background and margins not set

Comment (mauricioac) [reply app/forms/award_years/v2022/qavs/qavs_step5.rb:341]: o.O ok, leave it then, but not common to use both a parent and child class together (in general)

**Pull request merged**